right side with presets working. in general timepicker fully working

Drsorab opened the pull request:
-presets working
-date time picker fully working returning values for all time and date ranges

Drsorab committed:
right side with presets working. in general timepicker fully working

d-mo commented on timerange-picker.html:
```diff
@@ -348,8 +374,20 @@ <h2 style="left:30px;">Quick Ranges</h2>
-        console.log(datetime);
-
+        if(this.innerTo!="now"){
+          var df=new Date(this.innerFrom);
```
inconsistent indentation

Drsorab committed:
code was cleaned a bit
unessecary parts removed
innerfrom and innerTo added so as not to change from and to variables because that would trigger an ajax requests as soon as they were changed

Drsorab committed:
some of the code removed because it wasnt any longer necessesary.
clicking left and right  subtracts or adds 10 mins to the time range and makes a request for the specified time span.works properly
